pkg/resource/events-logger-config/alloy/events-logger.alloy.template

```diff
@@ -59,6 +59,11 @@ loki.write "default" {
 }
 
 {{- if .TracingEnabled }}
+otelcol.auth.basic "tracingCredentials" {
```

Comment:
Not sure about the name . I think alloy usually uses stuff like tracing_credentials?